Resources/public/js/main.js

```diff
@@ -13,33 +13,31 @@
             },
             options
         );
-        $(this.options.containerSelector).on('click', this.options.buttonSelector, this.clickHandler.bind(this));
+        $(this.options.containerSelector)
+            .find(this.options.buttonSelector)
+            .on('click',this.clickHandler.bind(this));
```

Comment:
Could you please come back to the previous implementation here. The difference is small, but previous version permit to handle AJAX loaded content (while the `find` version doesn't AFAIK).